font/grub2font.ksy

```diff
@@ -0,0 +1,100 @@
+meta:
+  id: grub2font
+  title: GRUB2 font
+  file-extension: pf2
+  tags:
+    - font
+  license: CC0-1.0
+  endian: be
+doc: |
+  Bitmap font format for the GRUB 2 bootloader.
+doc-ref: http://grub.gibibit.com/New_font_format
+seq:
+  - id: font_header
+    contents: ['FILE', 0, 0, 0, 4, 'PFF2']
+    size: 12
+  - id: font_sections
+    type: font_section
+    repeat: until
+    repeat-until: _.section_name == "DATA"
```

Comment:
Why and how is `data` so special ?

Reply:
It acts as a terminator. From the documentation:

"A marker that indicates the remainder of the file is data accessed via the character index (CHIX) section. When reading this font file, the rest of the file can be ignored when scanning the sections. "

Reply:
Can some words be added into the `doc` of that enum value?

Reply:
I added it near the repeat as I think it makes more sense there, but happy to move it.